e2e/tests/interchain_accounts/base_test.go

```diff
@@ -34,34 +37,46 @@ type InterchainAccountsTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+func (s *InterchainAccountsTestSuite) SetupTest() {
+	ctx := context.TODO()
+	chainA, chainB := s.GetChains()
+	relayer := s.SetupRelayer(ctx, nil, chainA, chainB)
+	s.SetChainsIntoSuite(chainA, chainB)
+	s.SetRelayerIntoSuite(relayer)
```

Comment:
we probably don't want to setup the relayer in every test. At the moment it looks like we have one test suite level relayer, but we are setting it up in every test. I think we should probably set up 1 relayer per test that is acting on a single channel. 

One issue that we might run into here, is that I think the default hermes config is watch all channels, so it might be easier for the first draft to do a single relayer set up that will watch all channels.

We can always tweak it in the future, if we go with that approach though, we'll want the relayer set up only a single time.

Reply:
sorry, this is incorrect, I thought `SetupTest` ran before each test but it runs before the test suite.

I think it makes sense for chain creation to exist here, maybe we would want to move channel and relayer creation into `BeforeTest(suiteName, testName string)`.